### PR TITLE
Update About page

### PR DESCRIPTION
**Pull request by matlegit**

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -4,28 +4,28 @@ title: About
...
+This is the official blog for the Kodethon technical assessment service.
+Kodethon's purpose is simple: <b>to help companies find better candidates and universities spend more time teaching</b>.
 
 Our Team:
-* Martin Velez, Ph.D. Student at University of California, Davis
-* Michael Yen, Undergraduate Student at University of California, Davis
-* Mathew Le, Undergraduate Student at University of California, Davis
+* Martin Velez, Ph.D., Cofounder
+* Michael Yen, Cofounder
+* Matt Le, Cofounder
```
Comment (matlegit) [inline about.md]: Add roles here?

File: about.md [hunk 2]
```diff
@@ -4,28 +4,28 @@ title: About
...
 * Kevin Yen, Undergraduate Student at University of California, Berkeley
```
Comment (matlegit) [inline about.md]: Add Bridget too?

File: about.md [hunk 3]
```diff
@@ -4,28 +4,28 @@ title: About
...
+* [https://support.kodethon.com/](https://support.kodethon.com/)
 * [https://kodethon.github.io/](https://kodethon.github.io/)
 
 
 <div style="text-align: center;border:1px solid
 lightgrey;background-color:white">
 <br>
 <p style="margin:0;color:#F39237;font-family:'Varela Round',
-sans-serif;font-size:1.5rem"><em>"Helping people code."</em></p>
+sans-serif;font-size:1.5rem"><em>"A better technical assessment"</em></p>
```
Comment (matlegit) [inline about.md]: Thoughts?

File: about.md (hunk 2 above)
Comment (matlegit) [reply about.md]: Katherine?

File: about.md (hunk 3 above)
Comment (Jvlythical) [reply about.md]: I'm not sure about the word "betteR". Alternatives: Technical assessment for real-world problems, Find better developers faster.

Comment (matlegit) [reply about.md]: What about the education offering?

**Comment by matlegit** [comment]
Also included a README change

**Pull request merged**